main.sh

```diff
@@ -14,9 +14,23 @@ GITHUB_REPO_OWNER=${GITHUB_REPOSITORY%%/*}
 
 phpcs_standard=''
 
-if [[ -f "$RTBOT_WORKSPACE/phpcs.xml" ]]; then
-    phpcs_standard="--phpcs-standard=$RTBOT_WORKSPACE/phpcs.xml"
-else
+defaultFiles=(
+  '.phpcs.xml'
+  'phpcs.xml'
+  '.phpcs.xml.dist'
+  'phpcs.xml.dist'
+)
+
+phpcsfilefound=1
```

Comment:
Should this be 0 at this point? as the file is not found yet. @mbtamuli

Reply:
Setting it to be explicitly 1. In case file is found, setting it to be zero. 

Just a convention followed because this is bash. https://stackoverflow.com/questions/2933843/why-0-is-true-but-false-is-1-in-the-shell